sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/Models/AlertSimulatorRequestBody.cs

```diff
@@ -14,27 +14,23 @@ namespace Microsoft.Azure.Management.Security.Models
     using System.Linq;
 
     /// <summary>
-    /// IoT sensor
```

Comment:
This file seems like a serious conflict!!! please check with IoT that this is fine

Reply:
Maybe it's just the git compare acting up, it is like it thinks that IotSensor was renamed to AlertSimulatorRequestBody

Reply:
yes this seem odd. 
IotSensor was deprecated and the AlertSumulatorRequestBody is a new addition so git created this "conflict".

Reply:
Doesn't seem like there is a real issue here other than this strange rename.
Those files are auto generated anyway